### ISSUE TITLE
Storybook doc site mobile navigation missing guide links

### ISSUE THREAD

**Issue by redonkulus** [storybookjs/frontpage#447]

**Comment by winkerVSbecks** [comment]
@redonkulus do you mean this sidebar? Yea, good catch. It's not visible on mobile. 

<img width="347" alt="CleanShot 2022-12-14 at 08 54 27@2x" src="https://user-images.githubusercontent.com/42671/207613365-a4ba96f7-e4c4-49b5-9f97-367c2f56f8fb.png">

**Comment by redonkulus** [comment]
Yes that is correct, the sidebar. Is this intentional? How do we navigate to the sidebar items otherwise?
